doc/Makefile

```diff
@@ -7,7 +7,8 @@ RST2HTML     = rst2html
 SPHINXOPTS   =
 SPHINXVER    = 0.5
 SPHINXBUILD  = sphinx-build
-PAPER        =
+# This one can also be set in the environment
+PAPER        ?= letter
```

Comment:
There was a typo so this PAPER Makefile variable had no effect.  Sphinx defaults to using 'letterpaper', hence the update. I think  it makes sense to allow also setting PAPER in environment not only command line hence the `?`.